src/page/bank-payment.page.ts

```diff
@@ -4,7 +4,7 @@ export class BankPaymentPage {
   private confirmOrderButton: ElementFinder;
 
   constructor () {
-    this.confirmOrderButton = $('#cart_navigation > button > span');
+    this.confirmOrderButton = $('.cart_navigation > .button > span');
```

Comment:
try to find a better locator, something that uses the name or title attribute for example, in that way, we can understand what element you are referring